Mustache/src/css/box.scss

```diff
@@ -0,0 +1,66 @@
+@import "colors";
+
+.box{
+    position: relative;
+    margin-bottom: 20px;
+    margin-left: 0px; 
+    width: 360px;
+    height: 242px;
+    &__options{
+        margin-top: 0;
+        display: flex;
+        flex-wrap: column;
+        &__price{
+            font-size: 14px;
+            font-weight: 900;
+        }
+        &__button{
+            color: $text-white;
+            margin-left: 25px;
+            &:hover{
+                font-size: 18px;
+            }
+        }
+    }
+    &__container-header{
+        margin-left: 0;
+        display: flex;
+        flex-wrap: column;
+        &__title{
+            font-size: 18px;
+            margin-left: 0;
+        }
+        &__logo{
+            position: absolute;
+            margin-top: 4px;
+            left: 270px;
+            width: 40px;
+            height: 40px;
+        }
+    }
+    &__information{
+        margin-top: 0;
+        display: flex;
+        flex-wrap: column;
+        &__text{
+            position: relative;
+            width: 172px;
+            height: 119px;
+            margin-top: 0;
+            padding-left:  15px;
+            text-align: justify;
+            font-size: 12px;
+        }
+        &__picture{
+            width: 126px;
+            height: 117px;
+        }
+    }
+    &__rayita{

```

Comment:
Be aware of not mixing spanish-english names. Stick with one (english preferable).

Reply:
And be sure that your variable names mean something, not just 'rayita'.